### PR TITLE
ref(ts): Refactor `<OrganizationActivity>`

### PR DESCRIPTION
**Pull request by billyvg**
* Refactors to typescript
* Move to lightweight routes
* Use AsyncView
* Refactor to use `emotion`

### CODE REVIEW

**Comment by billyvg** [comment]
This also changes the sentry logo from

image


to 

image

File: src/sentry/static/sentry/app/views/organizationActivity/activityFeed.jsx [hunk 1]
```diff
@@ -1,140 +0,0 @@
-import PropTypes from 'prop-types';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationActivity/activityFeed.jsx:1]: moved this into `index`

File: src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.tsx [hunk 2]
```diff
@@ -301,104 +299,127 @@ class ActivityItem extends React.Component {
...
+    const {className, item} = this.props;
 
     const avatar = item.user ? (
-      <UserAvatar user={item.user} size={36} className="activity-avatar" />
+      <UserAvatar user={item.user} size={36} />
     ) : (
-      <div className="activity-avatar avatar sentry">
-        <span className="icon-sentry-logo" />
-      </div>
+      <IconSentry size="36px" />
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.tsx:307]: Should this be a rounded icon like we use on discover query cards and issue activity?

File: src/sentry/static/sentry/less/organization.less [hunk 3]
```diff
@@ -1,126 +1,3 @@
-.organization-home {
```
Comment (markstory) [inline src/sentry/static/sentry/less/organization.less:1]: :clap:

**Review by markstory — APPROVED** [review]
Looks good from a read through.

File: src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.tsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.tsx]: Updated to re-use `<ActivityAvatar>`

**Pull request merged**